tst/shaka_scheme/system/base/unit-Environment.cpp

```diff
@@ -109,7 +109,22 @@ TEST(EnvironmentUnitTest, get_value_invalid_key_failure) {
 }
 
 TEST(EnvironmentUnitTest, contains) {
+  // Given: an environment and different symbols and values
+  Environment e(nullptr);
+  Symbol key1("x");
+  Symbol key2("y");
+  auto x_value = create_node(String("x test"));
 
```

Comment:
I added the unit test for contains. Thanks for catching that

Reply:
No problem. Thanks for the commit. I'll let Austin do the rebase and merge.